src/Bindings/LuaState.cpp

```diff
@@ -398,7 +398,7 @@ void cLuaState::Push(const cCraftingGrid * a_Grid)
 {
 	ASSERT(IsValid());
 
-	tolua_pushusertype(m_LuaState, (void *)a_Grid, "cCraftingGrid");
+	tolua_pushusertype(m_LuaState, reinterpret_cast<void *>(const_cast<cCraftingGrid*>(a_Grid)), "cCraftingGrid");
```

Comment:
This cast (and all the following ones) are malformed - add a space between the type and asterisk:
`<cCraftingGrid *>`